app.py

```diff
@@ -1,41 +1,52 @@
 import base64
-from functools import wraps
+import datetime
 import json
 import os
-import datetime
 import pathlib
 import re
 import sys
+from functools import wraps
 from tkinter import ALL
-from flask import Flask, abort, render_template, request, redirect, url_for, flash, session, jsonify, send_from_directory
-from bson import ObjectId
-from google_auth_oauthlib.flow import Flow
+
+import fitz
+import google.auth.transport.requests
 import requests
+from bson import ObjectId
+from flask import (
+    Flask,
+    abort,
+    flash,
+    jsonify,
+    redirect,
+    render_template,
+    request,
+    send_from_directory,
+    session,
+    url_for,
+)
 from google.oauth2 import id_token
-import google.auth.transport.requests
+from google_auth_oauthlib.flow import Flow
+from PIL import Image
 from pip._vendor import cachecontrol
-from Database import userdatahandler
 from werkzeug.utils import secure_filename
-import fitz  
-from PIL import Image
-
 
+from Database import userdatahandler
 from Database.admindatahandler import check_admin_available, create_admin, is_admin
 from Database.userdatahandler import (
-    create_user, 
+    create_user,
     delete_image,
-    get_currentuser_from_session, 
-    get_image_by_id, 
-    get_images_by_user, 
-    get_password_by_username, 
-    get_user_by_username, 
-    is_email_available, 
+    get_currentuser_from_session,
+    get_image_by_id,
+    get_images_by_user,
```

Comment:
Why is this file committed? I do not see any change.
Is it intentional to remove the end of line characters or was it a mistake?

If it is a mistake, can you remove this file from the commit and re-submit the pull request?

While Beehive is a small project with probably around only 4 people actively working on it, this is a best practice in the open source and software development in general: 
1. Don't change any file unless you really mean it or need it.
2. If you really need to change a file for reasons other than the one stated in the pull request, make it into a new pull request.